cinema/tests/test_movie_api.py

```diff
@@ -83,9 +83,7 @@ def test_get_movie(self):
         self.assertEqual(response.data["genres"][1]["name"], "Comedy")
         self.assertEqual(response.data["actors"][0]["first_name"], "Kate")
         self.assertEqual(response.data["actors"][0]["last_name"], "Winslet")
-        self.assertEqual(
-            response.data["actors"][0]["full_name"], "Kate Winslet"
-        )
+        self.assertEqual(response.data["actors"][0]["full_name"], "Kate Winslet")
```

Comment:
Ensure that the URL '/api/cinema/movies/' matches the actual URL pattern defined in your Django project. Hardcoding URLs can lead to maintenance issues if the URL patterns change. Consider using Django's reverse function to dynamically generate URLs based on the view names.